### ISSUE TITLE
[CONTENT]

### ISSUE THREAD

**Issue by Otto404** [dapr/docs#1167]
**Describe the issue**
Execution of the install script returns an error. The version is not installed. dapr is not installed.
The error-message is:

In Zeile:1 Zeichen:106
+ ... /dapr/cli/master/install/install.ps1; =[ScriptBlock]::Create(); invok ...
+                                                                  ~
Nach "(" wurde ein Ausdruck erwartet.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : ExpectedExpression

**URL of the docs**
https://v1-rc3.docs.dapr.io/getting-started/install-dapr-cli/

**Expected content**
dapr CLI v1.0-rc.4.  should be installed.

**Screenshots**

![image](https://user-images.githubusercontent.com/3367832/107079091-a64f6d80-67ef-11eb-8034-21a949b0387b.png)

**Comment by msfussell** [comment]
@Otto404  - Do you have German locale for Windows installed on your machine? This may be a localization issue. Could you try this on an English locale machine? We are not able to repo this.

**Comment by Otto404** [comment]
Yes, I have a German locale and: no I have no English local machine.
